guides/assets/stylesrc/_main.scss

```diff
@@ -56,8 +64,9 @@ body.guide {
     hyphenate-limit-last: always;
 
     margin: 0 0 0.75em 0; // Space after paragraph
-    text-align: left;
 
+    :where(body[dir="ltr"]) & { text-align: left; }
+    :where(body[dir="rtl"]) & { text-align: right; }
```

Comment:
We have moved these to the `html` tag, so we're not modifying the body anymore. Do you prefer to change the body instead?

It's a simple change to the JS to target the body, change line 111:

``` js
// from
mutation.target.dir = 
// to
document.querySelector("body").dir = 
```

For instance, I believe some of these changes broke a few things like the icon positions:

<details><summary>Details</summary>
<p>


![Screenshot 2024-04-23 at 11 00 51](https://github.com/rails/rails/assets/26328/953fbea3-d4a9-45eb-b74f-81db38610ee8)
![Screenshot 2024-04-23 at 11 01 02](https://github.com/rails/rails/assets/26328/fd65b65f-0e30-41d3-ab5d-44531c00d39b)

</p>
</details>